types/node-forge/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for node-forge 0.7.6
+// Type definitions for node-forge 0.7.7
```

Comment:
```suggestion
// Type definitions for node-forge 0.7.6
```

Reply:
https://www.npmjs.com/package/node-forge current v is 0.7.6

Reply:
Oops I wasn't thinking, for some reason I thought types were versions independently. Fixed.

Reply:
yeah, i did the same thing 🙃